src/main/kotlin/se/zensum/ktorSentryFeature/ReadHostname.kt

```diff
@@ -0,0 +1,28 @@
+package se.zensum.ktorSentry
+
+import java.io.File
+import java.net.InetAddress
+import java.nio.file.Files
+import kotlin.streams.toList
+
+internal fun hostname(): String? {
+    val envVarHostName: String? = System.getenv("HOSTNAME")
+    val envVarComputerName: String? = System.getenv("COMPUTERNAME")
+    val hostnameFile: String? = readHostnameFile()
+    val getHostName: String? = InetAddress.getLocalHost().hostName
```

Comment:
Should we have a fallback if none of these returns a decent value? Is that a possibility?

Reply:
It is certainly a possibility. I don't know if we should have one tough, you could consider all these options as a fallback, since the user may still set a host name when initiating the Sentry feature in the configuration block. If think not setting the host name will make it most clear that no host name was found or set by the user. What would be a good fallback value?